model/src/cfd.rs

```diff
@@ -1965,18 +1953,8 @@ pub fn calculate_short_liquidation_price(
         }
         ContractSymbol::EthUsd => {
             let initial_price = initial_price.to_u64();
-            let n_contracts = quantity.to_u64();
-            let multiplier = ETHUSD_MULTIPLIER;
-
-            let initial_margin =
-                quanto::calculate_initial_margin(initial_price, n_contracts, leverage, multiplier);
 
-            let liquidation_price = quanto::bankruptcy_price_short(
-                initial_margin,
-                n_contracts,
-                initial_price,
-                multiplier,
-            );
+            let liquidation_price = quanto::bankruptcy_price_short(initial_price, leverage);
```

Comment:
🙃 nit: maybe we can harmonize the names and order of parameters of these functions, given that `inverse::calculate_short_liquidation_price(leverage, initial_price)` is named different and the parameters are passed flipped.